tools/update_faq_markdown.py

```diff
@@ -34,9 +34,9 @@
 order: -9
 ---
 
-# FAQs (feat. Rowlet Owl)
```

Comment:
I can't accept the removal of your name in here, coz many of the answers are in first person.